docs-master/Model.md

```diff
@@ -93,25 +95,43 @@ class Post extends Model {
 }
 ```
 
-## Relation fields
+### Compound fields
```

Comment:
I think this `computed` is a widely recognizable term for this (Vue, MobX), although we must note the they aren't reactive in any way. So maybe:

```suggestion
### Derived fields
```